scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -96,6 +96,10 @@ SG_ State : 0|8@1+ (1,0) [0|255] "" LOGGER
 
 BO_ 306 FSM_AIR_SHUTDOWN: 0 FSM
 
+BO_ 307 FSM_FLOW_METER: 8 FSM
+SG_ Primary_flow_meter_flow_rate : 0|32@1+ (1,0) [0|0] "" LOGGER 
```

Comment:
Min/max values aren't set

Reply:
This was the only way that I could send `NaN` on CAN. It displays the previous value before the `NaN` was sent when we set a limit.

Reply:
Right but this still won't work when we use the periodic CAN signals setters because it won't pass the min/max value checks?

e.g.
```
    if (App_CanMsgs_foobar_is_in_range(value) == true)
    {
        can_tx_interface->periodic_can_tx_table.foobar = value;
    }
```

Reply:
The function that sets in in range looks like this so it will "pass" the value checks

```
bool App_CanMsgs_fsm_flow_meter_primary_flow_meter_flow_rate_is_in_range(float value)
{
    (void)value;

    return (true);
}
```

Reply:
Hmm doesn't seem great but i guess it's good enough of a solution for now